this makes Log() calls work! should create a log file called ... tbd
apparently has no effect!
I even tried with VBOX_LOG=+all.e.l.f.s.b and VBOX_LOG_DEST=file=/tmp/a.log and with VBOX_LOG_DEST=stdout
and with VBOX_RELEASE_LOG=

Index: /var/tmp/portage/app-emulation/virtualbox-modules-6.1.18/work/vboxdrv/VBox/log-vbox.c
===================================================================
--- .orig/vboxdrv/VBox/log-vbox.c
+++ mod/vboxdrv/VBox/log-vbox.c
@@ -703,7 +703,7 @@ RTDECL(PRTLOGGER) RTLogDefaultInit(void)
         RTLogFlags(pLogger, "enabled unbuffered");
         pLogger->fDestFlags |= RTLOGDEST_DEBUGGER | RTLOGDEST_STDOUT;
 # endif
-# if 0 /* vboxdrv logging - ATTENTION: this is what we're referring to guys! Change to '# if 1'. */
+# if 1 /* vboxdrv logging - ATTENTION: this is what we're referring to guys! Change to '# if 1'. */
         RTLogGroupSettings(pLogger, "all=~0 -default.l6.l5.l4.l3");
         RTLogFlags(pLogger, "enabled unbuffered tid");
         pLogger->fDestFlags |= RTLOGDEST_DEBUGGER | RTLOGDEST_STDOUT;
